lambda/voting/frontend/src/Components/Voting/Voting.js

```diff
@@ -1,11 +1,14 @@
 import React, { useState } from "react";
-import getLambdaVotingConsentToken from "../../utils/lambdaInvokeUtils";
+import getLambdaVotingConsentToken, {
+  getCandidature,
+} from "../../utils/lambdaInvokeUtils";
 import "./Voting.css";
 
 export const Voting = ({ sdk }) => {
   const [voterId, setVoterId] = useState("");
   const [asEmail, setAsEmail] = useState("");
   const [result, setResult] = useState("");
+  const [candidateDetails, setCandidateDetails] = useState("");
   const [loading, setLoading] = useState(false);
 
   let handleSubmit = async (event) => {
```

Comment:
there are 2 submit buttons. Which one is being handled by which button? You need to rename these so it makes sense to the user what's happening here.

Reply:
fixed